streaming/src/test/scala/org/apache/spark/streaming/StreamingContextSuite.scala

```diff
@@ -392,6 +393,29 @@ class StreamingContextSuite extends SparkFunSuite with BeforeAndAfter with TimeL
     assert(!sourcesAfterStop.contains(streamingSourceAfterStop))
   }
 
+  test("registering and de-registering of progressListener") {
```

Comment:
For a bug fix, we use JIRA ID prefix. Could you update like this?
```scala
- test("registering and de-registering of progressListener") {
+ test("SPARK-28709 registering and de-registering of progressListener") {
```

Reply:
Sure, updated.